### ISSUE TITLE
Keyboard shortcut causes multiple copies of About

### ISSUE THREAD

**Issue by skube** [storybookjs/storybook#10069]
**Describe the bug**
Using the keyboard shortcut <kbd>,</kbd> successive times in a row will open multiple copies of **About**.

**To Reproduce**

1. After starting up Storybook, go to any page
2. Using keyboard, hit the <kbd>,</kbd> three times
3. Now, in **About** panel, click ⓧ button in top-right corner

**Expected behavior**
- The **About** panel should close
- There should only be one **About** panel

**Actual behavior**
There are now three copies of **About**. One must click ⓧ button to close each copy.

**System:**

```
Environment Info:

  System:
    OS: macOS 10.15.3
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  Binaries:
    Node: 10.18.1 - ~/.nvm/versions/node/v10.18.1/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v10.18.1/bin/npm
  Browsers:
    Chrome: 80.0.3987.132
    Firefox: 73.0.1
    Safari: 13.0.5
  npmPackages:
    @storybook/addon-a11y: ^5.3.12 => 5.3.12
    @storybook/addon-actions: ^5.3.12 => 5.3.12
    @storybook/addon-docs: ^5.3.12 => 5.3.12
    @storybook/addon-essentials: ^5.3.12 => 5.3.12
    @storybook/addon-knobs: ^5.3.12 => 5.3.12
    @storybook/addon-links: ^5.3.12 => 5.3.12
    @storybook/addon-storyshots: ^5.3.12 => 5.3.12
    @storybook/addon-storysource: ^5.3.12 => 5.3.12
    @storybook/addons: ^5.3.12 => 5.3.12
    @storybook/react: ^5.3.12 => 5.3.12

```

**Comment by shilman** [comment]
@skube any interest in submitting a fix?

**Comment by skube** [comment]
Not really sure where to begin. I _think_ it may involve [shortcuts.tsx `onKeyDown` method](https://github.com/storybookjs/storybook/blob/f7fdd76e5c9fb1b10dda00ca00a28e70b120e6aa/lib/ui/src/settings/shortcuts.tsx#L171)?

**Comment by shilman** [comment]
cc @ndelangen @tmeasday

**Comment by tmeasday** [comment]
Probably just that we navigate here: https://github.com/storybookjs/storybook/blob/f7fdd76e5c9fb1b10dda00ca00a28e70b120e6aa/lib/api/src/modules/shortcuts.ts#L252-L254

and clicking the X does a "back"?

**Comment by ndelangen** [comment]
If we add a check on line 253.. are we already on `/settings/about`?

OR we can make the router smarter?

**Comment by vanessayuenn** [comment]
Hi! I am going through and cleaning up old issues in the repo. We just shipped a ton of improvements in Storybook 7.0, and I suspect this bug might be fixed already. Can you please try it out? If this issue persists with the latest version, please open a new bug report with a 7.x reproduction. Thank you so much! 🙏🏼

**Comment by skube** [comment]
Yup, you suspected correctly. It's not longer an issue. Thanks!
